src/wasm/wasm-binary.cpp

```diff
@@ -795,6 +795,33 @@ void WasmBinaryWriter::writeNames() {
   // TODO: label, type, and element names
   // see: https://github.com/WebAssembly/extended-name-section
 
+  // GC field names
+  if (wasm->features.hasGC()) {
+    std::vector<HeapType> namedTypes;
```

Comment:
Maybe `typesWithNamedFields`? That's kind of verbose, though :/

Reply:
Good point. How about `relevantTypes`?